src/core/agent/services/signifyNotificationService.ts

```diff
@@ -283,6 +283,38 @@ class SignifyNotificationService extends AgentService {
         return;
       }
     }
+    if (notif.a.r === NotificationRoute.MultiSigExn) {
```

Comment:
This is all OK for now but we will need to revisit this, and figure out how we clean up old notifications etc.